compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -209,6 +213,7 @@ private SymbolTable(CompilerContext context) {
         context.put(SYM_TABLE_KEY, this);
 
         this.names = Names.getInstance(context);
+        this.symbolTable = SymbolTable.getInstance(context);
```

Comment:
Can't we use `this` where the symbol table is needed instead of getting it from the context here? This doesn't have to be a global variable anyway, right?

Reply:
Changed.